src/integrationTest/groovy/wooga/gradle/node/GithubIntegrationSpec.groovy

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 Wooga GmbH
+ * Copyright 2018-2022 Wooga GmbH
```

Comment:
Please only update the headers for files that have been altered

Reply:
I thought of updating it since the file was written in 2020 but had copyright of 2018.

Reply:
I backdated the unmodified ones to 2020 (repo creation date).